evalml/pipelines/generate_pipeline.py

```diff
@@ -0,0 +1,58 @@
+from .binary_classification_pipeline import BinaryClassificationPipeline
```

Comment:
Nit-picking, but can we call this file `generated_pipelines` per all the classes?